Sources/SnapshotTesting/Strategy/Codable.swift

```diff
@@ -9,8 +9,24 @@ extension Strategy where A: Encodable {
   }
 
   public static func json(_ encoder: JSONEncoder) -> Strategy<A, String> {
-    return Strategy.lines.pullback { encodable in
+    var strategy = Strategy.lines.pullback { (encodable: A) in
       try! String(decoding: encoder.encode(encodable), as: UTF8.self)
     }
+    strategy.pathExtension = "json"
```

Comment:
seems we'd want json path extension right?

also we could bring in overture to make this nicer, but then feel like that complicates supporting cocoapods and Carthage.